### PR TITLE
Add vignette for count_punc

### PR DESCRIPTION
**Pull request by lynnwbl**

### CODE REVIEW

File: vignettes/count_punc.Rmd [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Background
+
+During Natural Language Processing (NLP), data scientists may want:
+
+- extract additional characteristics from text data 
+- engineer their own features using these additional characteristics 
+- train their models with their engineered features for better model results
+
+The `count_punc` function from the `textfeatureinfo` package is one of four functions that aids in this text data extraction process. Specifically, `count_punc` makes it easy to extract the number of punctuation inside of a piece of text.
```
Comment (Jacq4nn) [inline vignettes/count_punc.Rmd]:
```suggestion
The `count_punc` function from the `textfeatureinfo` package is one of four functions that aids in this text data extraction process. Specifically, `count_punc` makes it easy to extract the number of punctuation inside a piece of text.
```

File: vignettes/count_punc.Rmd [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Note: punctuation characters
+
+Please note that in this version of the `textfeatureinfor`, the following characters are considered punctuation characters.
+
+#### `` , ! " # $ % & ' ( ) * + - . / : ; < = > ? @ [ ] ^ _ ` { | } ~ ``
+
+## Quick example with one piece of text
+
+To understand the basics of the `count_punc` function, we can take a look at it's interaction with a simple text object.
```
Comment (Jacq4nn) [inline vignettes/count_punc.Rmd]:
```suggestion
To understand the basics of the `count_punc` function, we can take a look at its interaction with a simple text object.
```

**Review by Jacq4nn — CHANGES_REQUESTED** [review]
Hi Lynn! Great work. Just have 2 corrections for grammar

**Comment by lynnwbl** [comment]
Thanks Jacqueline, they have been fixed.

**Review by Jacq4nn — APPROVED** [review]

**Pull request merged**